source/photo.html

```diff
@@ -0,0 +1,155 @@
+                Соседи будут рады!
+              </p>
+              <p class="photos-list__likes"> Нравится: 666</p>
+            </div>
+          </li>
+          <li class="photos-list__item">
+            <a href="#"><img class="photos-list__picture" width="280" height="180" src="" alt="Фотография Бориса"></a>
+            <div class="photos-list__description">
+              <p class="photos-list__author">Борис <span class="photos-list__timestamp">(20 часов назад)</span></p>
+              <p class="photos-list__text">
+                Здесь могла быть ваша цитата о высоком и вечном.
+              </p>
+              <p class="photos-list__likes"> Нравится: 215</p>
+            </div>
+          </li>
+          <li class="photos-list__item">
+            <a href="#"><img class="photos-list__picture" width="280" height="180" src="" alt="Фотография Сергея"></a>
+            <div class="photos-list__description">
+              <p class="photos-list__author">Сергей <span class="photos-list__timestamp">(25 часов назад)</span></p>
+              <p class="photos-list__text">
+                Самая кубанская в мире!
+              </p>
+              <p class="photos-list__likes"> Нравится: 4<p>
+            </div>
+          </li>
+          <li class="photos-list__item">
+            <a href="#"><img class="photos-list__picture" width="280" height="180" src="" alt="Фотография Максима"></a>
+            <div class="photos-list__description">
+              <p class="photos-list__author">Максим <span class="photos-list__timestamp">(40 часов назад)</span></p>
+              <p class="photos-list__text">
+                Где снег-то?
+              </p>
+              <p class="photos-list__likes"> Нравится: 150</p>
+            </div>
+          </li>
+          <li class="photos-list__item">
+            <a href="#"><img class="photos-list__picture" width="280" height="180" src="" alt="Фотография Надежды"></a>
+            <div class="photos-list__description">
+              <p class="photos-list__author">Надежда <span class="photos-list__timestamp">(45 часов назад)</span></p>
+              <p class="photos-list__text">
+                Всем GM и взаимные лайки!
+              </p>
+              <p class="photos-list__likes"> Нравится: 666</p>
+            </div>
+          </li>
+        </ul>
+        <p class="photos__description">
+          Затмите их всех! Выкладывайте все, что накопилось в телефоне!
+        </p>
+      </section>
+
+      <section class="editor">
+        <img class="editor__photo" width="280" height="374" src="" alt="Фотография для редактирования">
+        <div class="editor__instruments">
+          <button class="editor__tools editor__tools--crop editor__tools--active">Обрезать</button>
+          <button class="editor__tools editor__tools--fill editor__tools--deactive">Заливка</button>
+          <button class="editor__tools editor__tools--contrast editor__tools--deactive">Контраст</button>
```

Comment:
```suggestion
          <button class="editor__tools editor__tools--fill">Заливка</button>
          <button class="editor__tools editor__tools--contrast">Контраст</button>
```

раз деактива больше, то он дефолтный стиль, а актив через мод делаешь